src/js/yeldaWebchatInjector.js

```diff
@@ -401,6 +401,19 @@ class YeldaChat {
       }
       return
     }
+
+    // Triggered from webchat to listen leave viewport event
+    if (event.data && event.data.event && event.data.event === 'listenLeaveViewport') {
+      this.pushNotificationId = event.data.pushNotificationId
+      this.listenLeaveViewport()
+      return
+    }
+
+    // Triggered from webchat to stop listening to the leave viewport event
+    if (event.data === 'abstainLeaveViewport' || event.message === 'abstainLeaveViewport') {
```

Comment:
Why can we have either event.data and event.message? Can't we only have one (preferably event.data since we use in every other checks) ?